basic-setup/tasks.md

```diff
@@ -0,0 +1,6 @@
+# Basic Setup
+1. Install Homebrew on you laptop and install git
```

Comment:
So I went to [brew website] (http://brew.sh/) and typed **/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ** in the command prompt.